### PR TITLE
fix: Add unique name to VideoPress e2e tests package

### PR DESCRIPTION
**Pull request by dcalhoun**
#### Changes proposed in this Pull Request:

The VideoPress e2e tests package shared a name with the starter-plugin e2e tests package. This collision caused [Metro build errors](https://buildkite.com/automattic/gutenberg-mobile/builds/3724#01826f9d-0386-455a-ba0a-4618a227ec05/204-831) for the mobile Gutenberg editor project. Providing a unique name for each package addresses the collision.

Colliding package names:

https://github.com/Automattic/jetpack/blob/5e97040ad217da10434cbe6f0ba31b0b499388a6/projects/plugins/videopress/tests/e2e/package.json#L2
https://github.com/Automattic/jetpack/blob/5e97040ad217da10434cbe6f0ba31b0b499388a6/projects/plugins/starter-plugin/tests/e2e/package.json#L2

Metro build error: 

```
jest-haste-map: Haste module naming collision: jetpack-starter-e2e-tests
  The following files share their name; please adjust your hasteImpl:
    * <rootDir>/jetpack/projects/plugins/starter-plugin/tests/e2e/package.json
    * <rootDir>/jetpack/projects/plugins/videopress/tests/e2e/package.json

Failed to construct transformer:  DuplicateError: Duplicated files or mocks. Please check the console for more info
    at setModule (/workdir/gutenberg/node_modules/jest-haste-map/build/index.js:543:17)
    at workerReply (/workdir/gutenberg/node_modules/jest-haste-map/build/index.js:614:9)
    at runMicrotasks (<anonymous>)
    at processTicksAndRejections (internal/process/task_queues.js:95:5)
    at async Promise.all (index 154227)
    at async /workdir/gutenberg/node_modules/jest-haste-map/build/index.js:426:22
    at async Function.load (/workdir/gutenberg/node_modules/metro/src/node-haste/DependencyGraph.js:143:36) {
  mockPath1: 'jetpack/projects/plugins/starter-plugin/tests/e2e/package.json',
  mockPath2: 'jetpack/projects/plugins/videopress/tests/e2e/package.json'
}
error Duplicated files or mocks. Please check the console for more info.
Error: Duplicated files or mocks. Please check the console for more info
```

#### Other information:

- [x] Have you written new tests for your changes, if applicable?
- [x] Have you checked the E2E test CI results, and verified that your changes do not break them?

#### Jetpack product discussion

p1659735307022759-slack-C6UJ0KRKQ

#### Does this pull request change what data or activity we track or use?

No.

#### Testing instructions:

n/a

### CODE REVIEW

File: projects/plugins/videopress/tests/e2e/package.json
Comment (geriux) [inline projects/plugins/videopress/tests/e2e/package.json]: It looks like other e2e tests have `jetpack-` before the name of the test, what do you think if we change it to:

```suggestion
	"name": "jetpack-videopress-e2e-tests",
```
I don't have much experience with this repo but thought I'd mention it. Let me know, thanks!

Comment (dcalhoun) [reply projects/plugins/videopress/tests/e2e/package.json]: @geriux good idea. I have committed the rename to align with existing convention. Let me know if you think this is ready for a Jetpack Crew team member review. 🙇🏻

Comment (geriux) [reply projects/plugins/videopress/tests/e2e/package.json]: Awesome! Thank you! 

> Let me know if you think this is ready for a Jetpack Crew team member review. 🙇🏻

Yup 👍

**Review by geriux — APPROVED** [review]
LGTM!

**Comment by dcalhoun** [comment]
Thank you, Gerardo! Marking this as ready for review by the Jetpack Crew now.

Note for the Crew: I don't have commit access for WP.com, so would be grateful if you could handle committing to WP.com for me too. 🙇🏻

**Review by samiff — APPROVED** [review]

**Comment by samiff** [comment]
@dcalhoun 

> Note for the Crew: I don't have commit access for WP.com

This PR touched `projects/plugins/videopress` files only, which isn't synced to WPcom.

> This collision caused [Metro build errors](https://buildkite.com/automattic/gutenberg-mobile/builds/3724#01826f9d-0386-455a-ba0a-4618a227ec05/204-831) for the mobile Gutenberg editor project.

This PR makes sense to have the unique name, though I'm not familiar with the E2E tests. Let us know if there is anything blocking after having this merged still 👍

**Comment by dcalhoun** [comment]
Thank you, @samiff. 🎉 I appreciate the help merging this and the additional information about `packages/plugins/videopress`. The Metro builds are now working after this merge. 👍🏻

**Pull request merged**
